cmd/gangue/gangue.go

```diff
@@ -126,13 +130,15 @@ func run(cmd *cobra.Command, args []string) {
 	}
 
 	// Download the file and verify it (unless disabled)
-	if verify {
-		err = sdk.UpdateSignedFile(output, source, client, gpgKeyFile)
-		if err == nil && !keepSig {
-			err = os.Remove(output + ".sig")
+	for i, err = 0, fmt.Errorf("dummy"); i <= retry && err != nil; i++ {
```

Comment:
Any reason not to use the `util.Retry` helper?

Reply:
I don't think its wait time is useful here.  The HTTP failure responses seem to happen one at a time, not through some period of downtime.

Reply:
Would it not make sense to just use a duration of 0 in that case?